test/utils/fizzy_engine.cpp

```diff
@@ -51,11 +51,17 @@ bool FizzyEngine::instantiate()
 
 bytes_view FizzyEngine::get_memory() const
 {
+    if (!m_instance.memory)
+        return {};
+
     return {m_instance.memory->data(), m_instance.memory->size()};
 }
 
 void FizzyEngine::set_memory(bytes_view memory)
 {
+    if (!m_instance.memory)
```

Comment:
The other implementations only have a check for `!memory.empty()`.

Reply:
Why? I think I'll add this check to other engines, too.

Reply:
Because I think it assumes (and this is how bench is implemented) that if `get_memory` failed then it aborts.

Reply:
In context of benchmarking, I'm not sure returning `bool` is useful. It can only help when adding new benchmarks to detect incompatibilities with this API. But that would be noticed anyway by comparing final results.

In this PR, I would avoid fixing issues with `.set_memory()` provided that current benchmarks works. Maybe we should rework the interface instead of fixing current one.

Reply:
I think we can leave in this PR just assertions in `set_memory` that memory exists.